python_modules/dagster/dagster/components/core/defs_module.py

```diff
@@ -48,6 +50,7 @@ class ComponentFileModel(BaseModel):
 
     type: str
     attributes: Optional[Mapping[str, Any]] = None
+    template_vars_module: Optional[str] = None
```

Comment:
thoughts on generalizing this to context preparation / pre-load hook ?

Reply:
Further extending the ability to override or customize component behavior without having to subclass it. 

The other thing in my head is a conversation with Ben the other day from the snippet tests trying think of how to setup a bunch of mocking across a process hop that loaded components where something like could have been abused.